script.js

```diff
@@ -1,28 +1,37 @@
 document.addEventListener("DOMContentLoaded", function () {
+	setTimeout(() => {
+		let liAppear = document.querySelectorAll('.appear');
+		for (let i = 0; i < liAppear.length; i++) {
+			liAppear[i].classList.remove('appear');
+			console.log('usuwanie');
+		}
+	}, 1);

```

Comment:
@PrzemekBartecki I suggest defining constant instead of using magic number (it's anti pattern and you can read more about it e.g. here: https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544). You could call this variable e.g. ANIMATION_DELAY